openedx/features/caliper_tracking/transformers/problem_transformers.py

```diff
@@ -111,3 +111,36 @@ def problem_save(current_event, caliper_event):
         'ip': current_event['ip']
     })
     return caliper_event
+
+
+def save_problem_success(current_event, caliper_event):
+    """
+        The server emits save_problem_success events when a problem is saved successfully.
```

Comment:
Please fix the whitespace in the docstring.

Reply:
done in the latest commit.